components/site/index.css

```diff
@@ -131,8 +131,19 @@ governing permissions and limitations under the License.
 
 .spectrum-Site-logo {
   margin-right: var(--spectrum-global-dimension-size-175);
-  width: var(--spectrum-global-dimension-size-400);
-  color: rgb(255, 2, 1) !important;
+  color: rgb(225, 37, 27) !important;
```

Comment:
maybe there is a different solution that does not require the !important bangs in Spectrum CSS?